src/api/useTestCaseServiceApi.ts

```diff
@@ -142,6 +142,20 @@ export class TestCaseServiceApi {
     return response.data;
   }
 
+  async exportTestCases(measureId: string, signal): Promise<Blob> {
+    const response = await axios.get(
+      `${this.baseUrl}/measures/${measureId}/test-cases/exports`,
+      {
+        headers: {
+          Authorization: `Bearer ${this.getAccessToken()}`,
+        },
+        responseType: "blob",
```

Comment:
I believe this API should have a request body of the list of test case IDs for export, so in the case of exporting all test cases, would need to send all test case IDs back

Reply:
You can double check with @chubert-sb also as he wrote the stub and is actively developing the full API implementation